includes/core-changes/aspnetcore/3.0/http-cookie-samesite-defaults-change.md

```diff
@@ -0,0 +1,42 @@
+### HTTP: Some cookie SameSite defaults changed to None
+
+`SameSite` is an option for cookies that can help mitigate some Cross-Site Request Forgery (CSRF) attacks. When this option was initially introduced, inconsistent defaults were used across various ASP.NET Core APIs. The inconsistency has led to confusing results. As of ASP.NET Core 3.0, these defaults are better aligned. This is an opt-in feature on a per-component basis.
```

Comment:
This description seems out of sync with the following sections.  It cites "inconsistent defaults" but **Old behavior** and **Reason for change** only mention one "old behavior" default.

Reply:
Nice catch! I'll get that fixed.